app/_locales/en/messages.json

```diff
@@ -1958,7 +1958,7 @@
     "message": "Sign"
   },
   "signNotice": {
-    "message": "Signing this message can have \ndangerous side effects. Only sign messages from \nsites you fully trust with your entire account.\n This dangerous method will be removed in a future version. "
+    "message": "We are unable to estimate the result of this signature. This signature could potentially perform any operation on your account's behalf, including granting complete control of your account and all of its assets to the requesting site. Only sign this message if you know what you're doing or completely trust the requesting site."
```

Comment:
minor proposed change to copy : `We are unable to **predict** the result of this signature...`

Reply:
++